client/my-sites/plugins/plugin-details-sidebar/index.jsx

```diff
@@ -87,7 +91,7 @@ const PluginDetailsSidebar = ( {
 				id="support"
 				icon={ { src: support } }
 				title={ translate( 'Support' ) }
-				description={ translate( 'Handled by WooCommerce.' ) }
+				description={ translate( 'Live chat support 24x7.' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'Live chat support 24X7' )` *ES Score: 14*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/plugin_usps)